tests/python_client/requirements.txt

```diff
@@ -28,8 +28,11 @@ pytest-parallel
 pytest-random-order
 
 # pymilvus
-pymilvus==2.6.0rc119
-pymilvus[bulk_writer]==2.6.0rc119
+pymilvus==2.6.0rc123
+pymilvus[bulk_writer]==2.6.0rc123
+
+# for protobuf
+protobuf==5.27.2
```

Comment:
Does this dependency need to be installed additionally?

Reply:
the default version is 6.x, which is too high for pymilvus, and it keeps printing warning. So i downgrade it.